src/supla/sensor/AHT.h

```diff
@@ -0,0 +1,145 @@
+Skip to content
```

Comment:
Please cleanup this file and fix cpplint issues reported here: https://ci.supla.org/SUPLA/supla-device/217/1/15